pkg/observable/channel/map.go

```diff
@@ -0,0 +1,30 @@
+package channel
+
+import (
+	"context"
+
+	"pocket/pkg/observable"
+)
+
+func Map[S, D any](
```

Comment:
Why do we need this? #PUC

Also, consider `MapObservable` but not a strong stance.

Reply:
:pray: godoc comment added. :raised_hands: 

I think adding "observable" would be inconsistent as this isn't an observable implementation itself. It uses the `channel.NewObservable()` constructor for the new observable that it returns.

Reply:
Sgtm!